docs/2_2_common_mechanisms.adoc

```diff
@@ -1160,7 +1160,30 @@ Both functions have the same arguments:
 * `nSensitivity` contains the length of `sensitivity`.
 
 _[Note that array variables will be serialized, so `nSeed` is only equal to `nKnowns` in the case of directional derivatives (resp., equal to `nUnknowns` in the case of adjoint derivatives) if all value references of `knowns` (resp., `unknowns`) point to scalar variables._
-_Likewise `nSensitivity` is only equal to `nUnknowns` (resp., `nKnowns`) if all value references of `unknowns` (resp., `knowns`) point to scalar variables.]_
+_Likewise `nSensitivity` is only equal to `nUnknowns` (resp., `nKnowns`) if all value references of `unknowns` (resp., `knowns`) point to scalar variables._
+
+_Note that it is the partial derivative and not the total derivative that is considered here._
+_When computing the partial derivative with respect to the independent variable_ latexmath:[t] _of the function_
```

Comment:
Isn't it always the partial derivative and not the total derivative?
Mentioning the independent variable explicitly here is confusing. It could be interpreted that just for the independent variable (i.e., time) it is the partial derivative otherwise it is the total derivative.

Reply:
Yes its not only valid for time